File: src/routes/tokenRoute.js
const routering = require('express').Router();
const getTokenController = require('../controllers/tokenController')

routering.get('/getToken',(req,res)=>{
    getTokenController.getTokenJira(req,res);
})

routering.post('/update',(req,res)=>{
    getTokenController.updateDate(req,res)
})

module.exports = routering

File: src/routes/tokenRoute.py
from flask import Blueprint, request, jsonify
from controllers.tokenController import getTokenJira, updateDate

routering = Blueprint('routering', __name__)

@routering.route('/getToken', methods=['GET'])
def get_token_route():
    return getTokenJira(request, jsonify)

@routering.route('/update', methods=['POST'])
def update_route():
    return updateDate(request, jsonify)
